Assets/Scripts/ssjAug2018/DeilveryTarget/DeliveryTarget.cs

```diff
@@ -0,0 +1,54 @@
+﻿using pdxpartyparrot.Core.Actors;
+
+using UnityEngine;
+
+// TODO: Lots. Basic class implementation so I can reference this in the manager
+namespace pdxpartyparrot.ssjAug2018.DeliveryTargets
+{
+    [RequireComponent(typeof(Collider))]
+    [RequireComponent(typeof(AudioSource))]
+    public sealed class DeliveryTarget : NetworkActor {
+
+        /* Unsure if this is going to get animations.
+        #region Animations
+        [Header("Animations")]
+
+        [SerializeField]
+        private Animator _animator;
+        #endregion
+        */
+
+        public int DeliveriesRequired;
+
+        private AudioSource _audiosource;
+
+        #region Unity Lifecycle
```

Comment:
don't indent regions

Reply:
Missed one. VS is indenting automatically :( Is there a setting for that you might know offhand?

Reply:
yeah I've had that one happen as well, I'm not sure which setting does it. it's not one of the usual auto-formatters I don't think.